sdk/compute/armcompute/example_availabilitysets_test.go

```diff
@@ -9,7 +10,7 @@ import (
 	"context"
 	"log"
 
-	"github.com/Azure/azure-sdk-for-go/sdk/armcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/arm"
 	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
 	"github.com/Azure/azure-sdk-for-go/sdk/compute/armcompute"
 	"github.com/Azure/azure-sdk-for-go/sdk/to"
```

Comment:
This import should be replaced with `github.com/Azure/azure-sdk-for-go/sdk/azcore/to`